style.css

```
@@ -0,0 +1,446 @@
+* {
+	box-sizing: border-box;
+}
+
+body {
+	margin: 0;
+	font-family: Pretendard;
+}
+
+a {
+	text-decoration: none;
+	cursor: pointer;
+}
+
+/* Header */
+header {
+	display: flex;
+	align-items: center;
+	background-color: #ffffff;
+	height: 70px;
+	position: fixed;
+	top: 0;
+	left: 0;
+	width: 100%;
+	z-index: 1;
+	border-bottom: 1px solid #dfdfdf;
+}
+
+.header_wrap {
+	width: 1120px;
+	height: 51px;
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	margin: 0 auto;
+}
+
+.logo {
+	width: 153px;
+	height: 51px;
+}
+
+.login {
+	display: inline-flex;
+	justify-content: center;
+	align-items: center;
+	width: 128px;
+	height: 48px;
+	padding: 12px 23px;
+	background-color: #3692ff;
+	color: #f3f4f6;
+	border-radius: 8px;
+	font-size: 16px;
+	font-weight: 600;
+	transition: all 300ms ease;
+}
+
+.login:hover {
+	background-color: #1967d6;
+}
+/* Header */
```

💊 제안
Header 관련 css의 시작인지, 끝인지 알기 모호하기 때문에, 주석을 각 영역 상하단에 모두 넣을 때는 아래처럼 하시는 것이 구분하기 좋습니다.

```suggestion
/* End: Header */
```

이렇게 하시면 상단의 주석은 `/* Start: Header */`가 되겠죠?